src/python/grpcio_tests/tests/unit/_rpc_test.py

```diff
@@ -191,6 +191,10 @@ def setUp(self):
 
     self._channel = grpc.insecure_channel('localhost:%d' % port)
 
+  def tearDown(self):
+    self._server.stop(None)
+    self._server_pool.shutdown()
```

Comment:
Please explicitly pass `wait=True` here. We want to be clear that if something prevents the thread pool from shutting down, this is where the test should hang.